src/main/java/uk/gov/companieshouse/company_appointments/controller/CompanyAppointmentController.java

```diff
@@ -76,7 +76,9 @@ public ResponseEntity<Void> patchCompanyNameStatus(
         try {
             companyAppointmentService.patchCompanyNameStatus(companyNumber,
                     requestBody.getCompanyName(), requestBody.getCompanyStatus());
-            return ResponseEntity.ok().build();
+            return ResponseEntity.ok()
+                    .header(HttpHeaders.LOCATION, String.format("/company/%s/appointments", companyNumber))
```

Comment:
The endpoint to get the list of a appointments to a company is "/company/{company_number}/officers"